packages/combobox/src/Combobox.tsx

```diff
@@ -754,12 +754,11 @@ export function Combobox<M extends boolean>({
       e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
     ) => {
       if (!disabled) {
+        // Prevents triggering the setOpen function called by clicking anywhere within the input wrapper.
+        e.stopPropagation();
```

Comment:
Do we want to stop propagation here? Will a user potentially want to handle the click event?

Reply:
A user can with onClear. If we don't stopPropagation here then the event will bubble and trigger the function called when a user clicks on the wrapper, which forces the menu to open.